### ISSUE TITLE
Enable duplicate-id-aria Axe rule for Caseflow

### ISSUE THREAD

**Issue by alisan16** [department-of-veterans-affairs/caseflow#12881]
#12742 updated the Sniffybara and Axe versions that run accessibility scans in the codebase. In order to meet Section 508 compliance, the Caseflow codebase should run cleanly with all `section 508`, `wcag2aa` and `wcag2a` rules enabled. [Axe rule descriptions can be found here](https://github.com/dequelabs/axe-core/blob/develop/doc/rule-descriptions.md)

## Description
This task is for enabling the `duplicate-id-aria` rule: Ensures every id attribute value used in ARIA and in labels is unique

## Background/context/resources
Approximate incidence rates for each failed rule can be found in the table below. For the rules that have the most number of errors, it is likely a common UI component that is the issue. Check the caseflow-frontend-toolkit repo.

| Axe rule-id | # of Errors |
|------------|------------|
| aria-roles | 760 |
| definition-list | 47 |
| aria-required-parent | 22 |
| listitem | 17 |
| aria-allowed-attr | 14 |
| duplicate-id-aria | 8 |
| aria-input-field-name | 6  |
| aria-required-children | 6 |
| list | 4 |
| color-contrast | 2  |
| scrollable-region-focusable | 1 |
| duplicate-id-active | |

## Technical notes
1. To get started, remove the rule from the `rules` object in spec/support/VA-axe-run-configuration.json. Rules are enabled by default. You can also set `"enabled": true`.
2. Run `bundle exec rspec` to get a sense of where the failures are. This takes a long time to run, so feel free to cancel once a few errors pop up.
3. Investigate the components which are throwing the Sniffybara errors and fix them for compliance.
4. Once the `rspec` tests complete without failure, you've succeeded!
